tests/i18n/patterns/urls/default.py

```diff
@@ -10,7 +10,10 @@
     path("not-prefixed-include/", include("i18n.patterns.urls.included")),
     re_path(_(r"^translated/$"), view, name="no-prefix-translated"),
     re_path(
-        _(r"^translated/(?P<slug>[\w-]+)/$"), view, name="no-prefix-translated-slug"
+        _(r"^translated/(?P<slug>[\w-]+)/$"),
+        view,
+        {"extra": True},
```

Comment:
Can you changing this to:
```
{"slug": "default-slug"}, # Ensure that kwargs override extra
```
so we know that reversing this works properly (this also requires you to change `match.kwargs` to `match.captured_kwargs` above I think)

Reply:
Done!